projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -268,7 +268,11 @@ public function add_upgrades_menu( $plan = null ) {
 	public function add_options_menu() {
 		parent::add_options_menu();
 
-		add_submenu_page( 'options-general.php', esc_attr__( 'Security', 'jetpack' ), __( 'Security', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/security/' . $this->domain, null, 2 );
+		$show_security_nav_item = Jetpack_Plan::supports( 'security-settings' );
+
+		if ( $show_security_nav_item ) {
+			add_submenu_page( 'options-general.php', esc_attr__( 'Security', 'jetpack' ), __( 'Security', 'jetpack' ), 'manage_options', 'https://wordpress.com/settings/security/' . $this->domain, null, 2 );
```

Comment:
do you think it's possible and a good idea to split up this line?
```suggestion
			add_submenu_page(
				'options-general.php',
				esc_attr__( 'Security', 'jetpack' ),
				__( 'Security', 'jetpack' ),
				'manage_options',
				'https://wordpress.com/settings/security/' . $this->domain,
				null,
				2
			);
```

Reply:
I made the change here: https://github.com/Automattic/jetpack/pull/20384/commits/cb0bea62a8b1b6c8c50f413bcd64076a3f27ba2f

The other calls to `add-submenu_page` are still in that format, but I guess the readability is different here since we're wrapping it in a conditional.